owncloudApp/src/main/java/com/owncloud/android/usecases/UploadFileFromContentUriUseCase.kt

```diff
@@ -66,6 +67,7 @@ class UploadFileFromContentUriUseCase(
         val behavior: String,
         val uploadPath: String,
         val uploadIdInStorageManager: Long,
-        val wifiOnly: Boolean
+        val wifiOnly: Boolean,
+        val chargingOnly: Boolean
```

Comment:
Let's add [trailing commas ](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas)when possible to keep commits a little bit cleaner 👍